web/ui/react-app/src/pages/Config.tsx

```diff
@@ -42,6 +43,12 @@ const Config: FC<RouteComponentProps & PathPrefixProps> = ({ pathPrefix }) => {
       )}
     </>
   );
-};
+}
 
-export default Config;
+export default ({ pathPrefix }: RouteComponentProps & PathPrefixProps) => {
+  return (
+    <Fetch url={`${pathPrefix}/api/v1/status/config`}>
+      {(state: FetchState<YamlConfig>) => <Config {...state} />}
+    </Fetch>
+  )
+}
```

Comment:
Hm, does the linter not catch missing end-of-file newlines yet?

Reply:
Linter is fine, I just forgot to "npm run lint" before commit